frontend/src/languages/fr.json

```diff
@@ -543,7 +543,7 @@
   "label.button.remove.block": "Supprimer le Bloc",
   "label.button.remove.report": "Supprimer le Rapport",
   "pathology.label.block.number": "Numéro du Bloc",
-  "button.label.genarateReport": "Générer un rapport",
```

Comment:
I think you forgot to modify the corespong property in the en.json file and its actual usage